ch.html

```diff
@@ -92,22 +92,21 @@ <h1><a href="index.html"><font style="color:#ab2548;">/ </font>Alphatron competi
         <div class="inner">
             <h2>Menu</h2>
             <ul class="links">
-                <li><a href="index.html">Home</a></li>
-                <li><a href="#verseny">About</a></li>
-                <li><a href="https://verseny.alphatusa.hu/competition-rules">Rules</a></li>
-                <li><a href="team.html">Organisers</a></li>
-                <li><a href="#kapcsolat">Contact</a></li>
+                <li><a href="index.html">Startseite</a></li>
+                <li><a href="#verseny">Über Uns</a></li>
+                <li><a href="https://verseny.alphatusa.hu/competition-rules">Regeln</a></li>
+                <li><a href="team.html">Organisatoren</a></li>
+                <li><a href="#kapcsolat">Kontakt</a></li>
             </ul>
-            <a class="close" href="#">Close</a>
+            <a class="close" href="#">schliessen</a>
```

Comment:
Is it customary to have "schliessen" written like this on a button (by that I mean the lowercase first letter and ss instead of ß)

Reply:
Yes, that is customary. Also the Swiss Standard German always uses a double s (ss) instead of the eszett (ß).